tests/conftest.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import requests_mock as r_mock
+
+from shellhub import ShellHub
+
+
+@pytest.fixture(scope="function")
+def shellhub():
+    with r_mock.Mocker() as m:
+        # Mock the login URL
+        login_url = "http://localhost.shellhub/api/login"
```

Comment:
It would be better to create a variable `DOMAIN_URL` for `http://localhost.shellhub`
And then use it for better readibility on whole file, and use it like below
```suggestion
        login_url = f"{DOMAIN_URL}/api/login"
```